auction-server/src/opportunity/repository/models.rs

```diff
@@ -59,15 +59,15 @@ pub struct OpportunityMetadataSvmProgramLimo {
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub struct OpportunityMetadataSvmProgramWallet {
     #[serde_as(as = "DisplayFromStr")]
-    pub user_wallet_address:         Pubkey,
-    pub maximum_slippage_percentage: f64,
+    pub user_wallet_address:  Pubkey,
+    pub maximum_slippage_bps: u16,
 }
 
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 #[serde(tag = "program", rename_all = "lowercase")]
 pub enum OpportunityMetadataSvmProgram {
     Limo(OpportunityMetadataSvmProgramLimo),
-    Phantom(OpportunityMetadataSvmProgramWallet),
+    Swap(OpportunityMetadataSvmProgramWallet),
```

Comment:
We should make sure that there is no phantom opportunity inside the DB. Maybe add a task to backlog for it?

Reply:
do you mean that we don't currently have any opportunities of this sort saved in the db?